programs/ssl/dtls_server.c

```diff
@@ -79,7 +79,9 @@ int main( void )
 #include "mbedtls/error.h"
 #include "mbedtls/debug.h"
 #include "mbedtls/timing.h"
+
 #include "test/certs.h"
+#include "test/random.h"
```

Comment:
No longer used.

Reply:
Since I'm rebasing anyway in order to resolve conflicts, I'll amend the commit that made this unused to remove it too.